### PR TITLE
Dockerizing graphql api

### PR DESCRIPTION
**Pull request by alexjoybc**
# Description

This PR includes the following proposed change(s):

- Dockerizing GraphQl Api

run

docker-compose up efiling-graphql

access at [http://localhost:8090/graphql-ui/](http://localhost:8090/graphql-ui/)

image

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [x] Yes
- [ ] No

If Yes: Has Docker been updated accordingly?

- [x] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ The eFiling hub is a foundational component to enhance citizen experiences for t
 | ------------------- | -------------------------------------------- | -------------------------------------------------------- |
 | backend             | all server side services                     | [README](src/backend/README.md)                          |
 | efiling-api         | the main api for interating with the service | [README](src/backend/efiling-api/README.md)              |
-| efiling-worker      | process submitted documents                  | [README](src/backend/efiling-worker/README.md)           |
```
Comment (TayGov) [inline README.md:41]: 🥇 surprised that was still there!

File: docker-compose.yml [hunk 2]
```diff
@@ -109,6 +109,17 @@ services:
       - NAVIGATION_EXPIRYTIME=10
       - CLAMAV_HOST=clamav
       - CLAMAV_TIMEOUT=150000
+    networks:
+      - fisu-net
+
```
Comment (TayGov) [inline docker-compose.yml:114]: Should this not have a banner?

File: src/backend/Dockerfile.graphQlApi [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+WORKDIR /efiling-graphql
+
+# Build quarkus api
+
+RUN mvn package -Dquarkus.package.type=fast-jar
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.3 
+
+ARG JAVA_PACKAGE=java-11-openjdk-headless
```
Comment (TayGov) [inline src/backend/Dockerfile.graphQlApi:41]: This should be 8 for now. Unless you want to make the split. 
ARG JAVA_PACKAGE=java-1.8.0-openjdk-headless

**Review by TayGov — CHANGES_REQUESTED** [review]

File: src/backend/Dockerfile.graphQlApi (hunk 3 above)
Comment (alexjoybc) [reply src/backend/Dockerfile.graphQlApi:41]: good point, it is running with 11 now, I wonder if we should give it a try...

**Review by TayGov — APPROVED** [review]

**Pull request merged**